### PR TITLE
Update Icon

### LINKED ISSUE #110: New Icon
Current icon is really strange, because this is my first ever created icon.

Icon should describe project

s

### PR DESCRIPTION
**Pull request by jannuary**
Resolves #110 
let me know if i broke something

### CODE REVIEW

File: pkgs/com.github.qarmin.czkawka.desktop [hunk 1]
```diff
@@ -4,5 +4,7 @@ Terminal=false
 Exec=czkawka_gui
 Name=Czkawka
 Comment=Multi functional app to clean OS which allow to find duplicates, empty folders, similar files etc.
-Icon=icon
+Icon=com.github.quarmin.czkawka
```
Comment (qarmin) [inline pkgs/com.github.qarmin.czkawka.desktop]: `qarmin` instead `quarmin`

**Review by qarmin — CHANGES_REQUESTED** [review]

**Comment by qarmin** [comment]
Under what license do you want to share this icon?
In my opinion, the best is `CC-BY-4.0` https://creativecommons.org/licenses/by/4.0/deed.en

**Comment by jannuary** [comment]
@qarmin I think that should be fine.

**Review by qarmin — APPROVED** [review]

**Comment by qarmin** [comment]
Big thanks for this great icon!

**Pull request merged**